### ISSUE TITLE
Policy_violations don't include project_id in slack notification

### ISSUE THREAD

**Issue by Glennvd** [GoogleCloudPlatform/forseti-security#1160]
- Forseti 1.1.10
 - Scanner / notifier, slack webhook integration

Most scanners work great with the slack pipeline and show exactly the information we need to check up on a problem. However, I've noticed that the alerts with `type: policy_violations` don't include the project_id. 

Other scanners do include the id, and the project id can also be found in the csv thats uploaded to cloud storage. 

So the question is simple, can we include the project/organisation in the slack notification as well?

**Comment by blueandgold** [comment]
Hi there,

We have introduced something called a `full_name` in Forseti 2.0 that [will included in all the violations](https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-dev/google/cloud/forseti/scanner/audit/iam_rules_engine.py#L653), which will clarify where the iam policy comes from.

The `full_name` contains the entire gcp hierarchy above the resource, so that you will know which project the violation is for.

Examples of a ```full_name```:
```
organization/123/project/my_projecti/iam_policy/project/my_project/
organization/123/project/my_other_project/cloudsqlinstance/foo_sql/
```

You can [find out more about v2.0 here](https://groups.google.com/a/forsetisecurity.org/d/msg/discuss/-eBRNICY2t4/CBR1RJy0AgAJ), and beta testing will open next week, which we encourage you to try.

Please let me know if you have any other questions!

**Comment by blueandgold** [comment]
I am going to consider this issue closed, as it will be addressed when 2.0 is launched.  Thanks again for opening this issue.
